codetransformer/transformers/literals.py

```diff
@@ -42,3 +42,30 @@ def visit_STORE_MAP(self, instr):
 
         yield Instruction(ops.STORE_SUBSCR)
         # TOS  = m
+
+
+class overloaded_bytes(CodeTransformer):
+    """
+    Decorator that applies a callable to each bytes literal in the decorated
+    function.
+
+    Parameters
+    ----------
+    f : callable
+        A callable to be applied to each bytes literal in the decorated
+        function.
+    """
+
+    def __init__(self, f):
+        super().__init__()
+        self.f = f
+
+    def visit_consts(self, consts):
+        return super().visit_consts(
+            tuple(
+                self.visit_consts(const) if isinstance(const, tuple)
```

Comment:
do you mean default in `codetransformer`? Why, this doesn't make sense in lazy. I think that this is needed in this case and the float case.

You do need to look through frozensets though because of the peephole optimizer.

```python
In [1]: def f():
   ...:     return a in {1.0, 2.0}
   ...: 

In [2]: f.__code__.co_consts[3]
Out[2]: frozenset({1.0, 2.0})
```